### PR TITLE
drop inactive/disabled resources (SOFTWARE-3332)

### PR DESCRIPTION
**Pull request by edquist**
drop resources marked as inactve or disabled, which are not referenced
in any downtime files nor in gracc records from the last year.

the list of resource names dropped here is:

```
KISTI-NSDC-CE03
KISTI-NSDC-GUMS
UVA-sunfire
uva-cms-se
OSG_LIGO_MIT
OSG_LIGO_GUMS
UARK_ACE
UTA_DPCC
UTA_SWT2_TEST1
Engagement_VOMS
RENCI-Hatteras
RENCI-Blueridge
RENCI-Blueberry
Engage-Submit3-Frontend
RENCI-Engagement
TTU-TINGE
Tusker
Firefly
Firefly_SE
Firefly-3
Firefly-2
T2_Nebraska_Hadoop
T2_Nebraska_Hadoop3
GPN-HUSKER
T2_Nebraska_Hadoop2
T2_Nebraska_Storage
US-Nebraska Disabled
T2_Nebraska_OSG
red-gateway3-ITB
VT_OSG
Hampton_ATLAS_CE
Davis_CMS
OSG_US_ASU_OCTILLO
UNI-OSGEDU
DEPRECATE MSU LAT
DEPRECATED UM LAT BW
SBGrid-Harvard-East
SBGrid-Harvard
SBGrid-Harvard-Exp
WQCG-Harvard-OSG
NYSGRID-ALBANY-ATHENA
Bern ce01
Bern ce02
UTD-HEP
UTD-HEP2
CancerComputer_Miron_CE
HMS-Orchestra-Dev
GCVO_UIS_CE
Clemson-IT
Clemson-ciTeam
Clemson-Birdnest
GCVO_PUJ_CE_TEST
GCVO_PUJ_CE
gpnjayhawk
GPN_KSU_CLUSTER
Vanderbilt_Dev_SE
Vanderbilt_Perfsonar_BW
Vanderbilt_Perfsonar_Latency
NYSGRID-CUNY-GRID
NOvA_StashCache_Server
ASGC_OSG
GCVO_UAM_CE
OCI-NSF
isuhep
T3_US_OSU
OSG_US_OSU_COWBOY
OSG_TEST_CE
GCVO_UAC_CE
GRASE-NU-CARTMAN
T3_US_FIT
Purdue-Hadoop-SE2
Purdue-Steele
Purdue-RCAC
Purdue-CMS-Test
T2_Purdue_Perfsonar_Bandwidth
Purdue-CMS-SE
T2_Purdue_Perfsonar_Latency
Purdue-Rossmann
Purdue-Physics
Purdue-Caesar
UCLA_Saxon_Tier3
NWICG_NDCCL
SAMPA PerfSONAR Latency
SAMPA PerfSONAR Bandwidth
NYSGRID_RIT
LC-glcc
GCVO_UPB_CE
brown-cms-gen3
GCVO_UC_CE
UWMilwaukee
GCVO_UNIANDES_CE
GCVO_UCALDAS_CE
NCSA_TUNGSTEN
perfSONAR_IllinoisHEP
Tufts_ATLAS_Tier3
UIC_PHYSICS
FloridaAMUniversity_CE
OSG_LIGO_PSU
perfSONAR_GT_testbed_latency
BNL-LCG2
perfSONAR_OU
GLOW_MINIOSG_UCSB
GROW-ITB
GROW_VOMS
GROW-SE
GROW-PROD
GROW_GUMS
GROW_STOR
UIOWA-OSG-PROD
GRASE-NYU-BENCH
GRASE-CSE-MAGIC
GRASE-MARIST-nysgrid11
GRASE-SU-CLUSTER04
GRASE-ALBANY-NYS
GRASE-BINGHAMTON
GRASE-UR-NEBULA
GRASE-RIT-GCLUSTER
GRASE-HWI-IDUN
DukeT3_SQUID
DUKET3-SE
UCSDT2-SE
UCSanDiego-OSG-Validation-SE-SRM_v2
UCSandiegoOSG-Prod-SE
UCSD_GlideinWMS_Submit_1
UCSD_GlideinWMS_Submit_3
UCSD_GlideinWMS_Submit_2
CrossOSG-CE-UCSD
GCVO_UV_CE
GRASE-GENESEO-ROCKS
MCGILL_HEP
NERSC-Carver
NERSC-PDSFSRM
NERSC-Jacquard
NERSC-Franklin
NERSC-ITB
NERSC-Davinci
STAR-Bham
osg-ce
MWT2_IU
MWT2_IU_SE
IU_ATLAS_Tier2
Indiana_Physics_ATLAS
LTU_CCT
LTU_OSG
TAMU_BRAZOS
TAMU_BRAZOS_SRM
Baylor-Tier3-CE
Baylor-Tier3-SE
FNAL_WILSON_2
USCMS-FNAL-WC1-SE2
USCMS-FNAL-XEN
OSG_INSTALL_TEST_2
FIFE_SUBMIT_1
FIFE_SUBMIT_GPSN01
FIFE_UCSD_FRONTEND_01
FNAL_GPFARM
OSG-ReSS-PROD
gc1-ce
GRATIA-OSG-DAILY
GRATIA-OSG-ITB-PERM
FNAL_FERMIGRID_TEST
ITB-ReSS
FNAL-ITB-GIP-TEST1
FNAL_FERMIGRID_ITB
FNAL_FERMIGRID_ITB_SGE2
FNAL_FERMIGRID_ITB_PBS2
FNAL_GPGRID_1
FNAL_FAPL_ITB_SE
FNAL_IF_GRIDFTP_NOVA
FNAL_FERMIGRID
FNAL_DZEROOSG_4
FNAL_DZEROOSG_3
FNAL_DZEROOSG_2
FNAL_DZEROOSG_1
FNAL_GPGRID_OPP_3
FNAL_CDFOSG_0
FNAL_CDFOSG_1
FNAL_CDFOSG_2
FNAL_GPGRID_QUOTA_3
FNAL_GPGRID_3
FNAL_GPGRID_2
Gridplexus
Rice
GridUNESP_SPO
GridUNESP_SE
SPRACE-OSG-RSV
SPRACE_Perfsonar_BW
SPRACE_Perfsonar_LT
LONI_OSG1
STAR-WSU
UFlorida-SSERCA
perfSONAR_UF
UFlorida-PG
UFlorida-SEb
UFlorida-IHEPA
LIGO-CIT-ITB
LIGO_CIT_SE
LIGO_CIT
LIGO-CIT-VTB
CIT_CMS_perfSONAR
CIT_HEP_SE
BU_ATLAS_Tier2
NET2-BU
NET2-HU
ORNL_NSTG
FIU-PG
ANLASC Squid
ANLASC-ASCNFS
ANLASC-ASCHEAD
ANLASC-CE
PNNL_OSG-Misregistered
Belle_CE_2
PNNL_CE_1
PNNL_CE00
PNNL_SE
OSC_SuperB_gridftp
OSC_SuperB_CE
CHTC-glidein
CHTC and BATLAB OSG Mirror
GLOW-CHTC VOMS server
GLOW-ATLAS
GLOW-ATLAS-SRM
VDT-ITB-MADISON
VDT-ITB
Lehigh_piranha
LBNL_DSD_ITB
oliverlblgov
LBNL_VTB
UC_Teraport
UC_T2DEV_ITB
UC_ATLAS_MWT2
MWT2_UC
MWT2_X_SE
GCVO_UTB_CE
GCVO_UDEM_CE
```

### CODE REVIEW

**Comment by brianhlin** [comment]
To make parsing this list easier, could you separate the ones you've removed because they're marked as "disabled" vs the ones that were marked as "inactive"? If they're both inactive and disabled, put them in the disabled list

File: topology/Brown University/Brown-CMS/brown-cms.yaml [hunk 1]
```diff
@@ -1,58 +0,0 @@
-GroupDescription: Brown CMS T3 cluster
-GroupID: 346
-Production: true
-Resources:
-  brown-cms-gen3:
-    Description: Third generation CMS T3 at Brown
-    FQDN: brux11.hep.brown.edu
-    Active: false
```
Comment (matyasselmeci) [inline topology/Brown University/Brown-CMS/brown-cms.yaml]: Don't remove this one! They haven't been set up yet.

File: topology/University of Wisconsin/GLOW EDU/VDT-ITB.yaml [hunk 2]
```diff
@@ -1,71 +0,0 @@
...
-    FQDN: vdt-itb.cs.wisc.edu
-    ID: 307
-    Services:
-      CE:
-        Description: Compute Element
-        Details:
-          hidden: false
-    VOOwnership:
-      OSG: 100
-  VDT-ITB-MADISON:
```
Comment (matyasselmeci) [inline topology/University of Wisconsin/GLOW EDU/VDT-ITB.yaml]: Oh hey here's another one you shouldn't remove

**Comment by matyasselmeci** [comment]
I would like to go one step further than Brian and request the following:
- resources that are obviously gone (i.e. have something like "7/26 disabled" in the GroupDescription) should be removed in a separate PR
- resources that are not obviously gone should be in a PR with one commit per file so the changes can be reverted easily in case we find out we're wrong about something

File: topology/University of Wisconsin/GLOW EDU/VDT-ITB.yaml (hunk 2 above)
Comment (brianhlin) [reply topology/University of Wisconsin/GLOW EDU/VDT-ITB.yaml]: Errr, why not? Those resources don't reflect the existing ITB hosts

Comment (matyasselmeci) [reply topology/University of Wisconsin/GLOW EDU/VDT-ITB.yaml]: VDT-ITB-MADISON contains itb-ce1.chtc.wisc.edu (see line 62). The line I highlighted was misleading.

**Comment by edquist** [comment]
For now i've rebased this with the stuff from the brown-cms and VDT-ITB resource groups restored....

If you'd like to wait for it, i can work on breaking this up into separate PRs as you requested above...

**Comment by brianhlin** [comment]
I think Mat was asking for 1 PR with 1 commit per removed resource, not separate PRs

**Comment by matyasselmeci** [comment]
2 PRs:
- one with obvious removals
- one with non-obvious removals, one commit per file

**Comment by matyasselmeci** [comment]
Obvious: contains a GroupDescription or a Description saying something like:

- "disabling empty resource group"
- "out of service"
- "Deactivated"
- "disabled 1/26"
- "De-commissioned"
- "Rob confirmed to Arvind that this is a de-activated resource. 2008-06-06"
-  "I didn't register this resource and it doesn't show up in my Resource view but I can change the description of the new Resource Group entry.  What is going on. Is there something wrong with the privileges?"

Non-obvious: doesn't contain anything like the above.

**Comment by brianhlin** [comment]
Let's deal with those in a second pass.

**Comment by matyasselmeci** [comment]
OK.

**Comment by edquist** [comment]
@brianhlin, here are the lists, based on the current `master`, of disabled resources, followed by enabled but inactive resources:

*disabled:*
```
KISTI-NSDC-CE03
KISTI-NSDC-GUMS
UVA-sunfire
uva-cms-se
OSG_LIGO_MIT
OSG_LIGO_GUMS
UARK_ACE
UTA_DPCC
Engagement_VOMS
RENCI-Hatteras
RENCI-Blueridge
RENCI-Blueberry
Engage-Submit3-Frontend
TTU-TINGE
Tusker
Firefly
Firefly_SE
Firefly-3
Firefly-2
T2_Nebraska_Hadoop
T2_Nebraska_Hadoop3
GPN-HUSKER
T2_Nebraska_Hadoop2
T2_Nebraska_Storage
US-Nebraska Disabled
T2_Nebraska_OSG
red-gateway3-ITB
VT_OSG
Hampton_ATLAS_CE
Davis_CMS
UNI-OSGEDU
DEPRECATE MSU LAT
DEPRECATED UM LAT BW
SBGrid-Harvard-East
SBGrid-Harvard
SBGrid-Harvard-Exp
WQCG-Harvard-OSG
NYSGRID-ALBANY-ATHENA
Bern ce01
Bern ce02
UTD-HEP
UTD-HEP2
CancerComputer_Miron_CE
HMS-Orchestra-Dev
FZU_NOVA
Clemson-IT
Clemson-ciTeam
Clemson-Birdnest
gpnjayhawk
GPN_KSU_CLUSTER
Vanderbilt_Dev_SE
Vanderbilt_Perfsonar_BW
Vanderbilt_Perfsonar_Latency
NYSGRID-CUNY-GRID
ASGC_OSG
OCI-NSF
isuhep
T3_US_OSU
GRASE-NU-CARTMAN
T3_US_FIT
Purdue-Hadoop-HTCE
Purdue-Hadoop-SE2
Purdue-Hadoop-SE
Purdue-Steele
Purdue-RCAC
Purdue-CMS-Test
T2_Purdue_Perfsonar_Bandwidth
Purdue-CMS-SE
T2_Purdue_Perfsonar_Latency
Purdue-Hansen
Purdue-Rossmann
Purdue-Carter
Purdue-Physics
Purdue-Caesar
UCLA_Saxon_Tier3
NWICG_NDCCL
NWICG_NotreDame
SAMPA PerfSONAR Latency
SAMPA PerfSONAR Bandwidth
NYSGRID_RIT
LC-glcc
UWMilwaukee
NCSA_TUNGSTEN
Tufts_ATLAS_Tier3
UIC_PHYSICS
FloridaAMUniversity_CE
OSG_LIGO_PSU
BNL-LCG2
OU_OSCER_CONDOR
perfSONAR_OU
GLOW_MINIOSG_UCSB
GROW-ITB
GROW_VOMS
GROW-SE
GROW-PROD
GROW_GUMS
GROW_STOR
UIOWA-OSG-PROD
GRASE-NYU-BENCH
GRASE-CSE-MAGIC
GRASE-MARIST-nysgrid11
GRASE-SU-CLUSTER04
GRASE-ALBANY-NYS
GRASE-BINGHAMTON
GRASE-UR-NEBULA
GRASE-RIT-GCLUSTER
GRASE-HWI-IDUN
DukeT3_SQUID
DUKET3-SE
UCSDT2-SE
UCSanDiego-OSG-Validation-SE-SRM_v2
UCSandiegoOSG-Prod-SE
CrossOSG-CE-UCSD
GRASE-GENESEO-ROCKS
MCGILL_HEP
NERSC-Carver
NERSC-PDSFSRM
NERSC-Jacquard
NERSC-Franklin
NERSC-ITB
NERSC-Davinci
STAR-Bham
MWT2_IU_SE
IU_ATLAS_Tier2
Indiana_Physics_ATLAS
LTU_CCT
LTU_OSG
TAMU_BRAZOS
TAMU_BRAZOS_SRM
Baylor-Tier3-CE
Baylor-Tier3-SE
USCMS-FNAL-XEN
OSG_INSTALL_TEST_2
FIFEBATCH2
FIFE_SUBMIT_1
FIFE_SUBMIT_GPSN01
FIFE_UCSD_FRONTEND_01
FNAL_GPFARM
OSG-ReSS-PROD
gc1-ce
GRATIA-OSG-DAILY
GRATIA-OSG-ITB-PERM
FNAL_FERMIGRID_TEST
ITB-ReSS
FNAL-ITB-GIP-TEST1
FNAL_FERMIGRID_ITB
FNAL_FERMIGRID_ITB_SGE2
FNAL_FERMIGRID_ITB_PBS2
FNAL_GPGRID_1
FNAL_FAPL_ITB_SE
FNAL_IF_GRIDFTP_NOVA
FNAL_FERMIGRID
FNAL_DZEROOSG_4
FNAL_DZEROOSG_3
FNAL_DZEROOSG_2
FNAL_DZEROOSG_1
FNAL_CDFOSG_4
FNAL_GPGRID_OPP_3
FNAL_CDFOSG_0
FNAL_CDFOSG_1
FNAL_CDFOSG_2
FNAL_CDFOSG_3
FNAL_GPGRID_QUOTA_3
FNAL_GPGRID_5
FNAL_GPGRID_4
FNAL_GPGRID_3
FNAL_GPGRID_2
UERJ_CE
Gridplexus
Rice
GridUNESP_SPO
GridUNESP_SE
SPRACE-OSG-RSV
SPRACE_Perfsonar_BW
SPRACE_Perfsonar_LT
LONI_OSG1
STAR-WSU
UFlorida-SSERCA
perfSONAR_UF
UFlorida-PG
UFlorida-SE
UFlorida-SEb
UFlorida-IHEPA
LIGO-CIT-ITB
LIGO_CIT_SE
LIGO_CIT
LIGO-CIT-VTB
CIT_CMS_perfSONAR
CIT_HEP_SE
BU_ATLAS_Tier2
ORNL_NSTG
FIU-PG
ANLASC-ASCNFS
ANLASC-ASCHEAD
ANLASC-CE
PNNL_OSG-Misregistered
Belle_CE_2
PNNL_CE_1
PNNL_SE
OSC_SuperB_gridftp
OSC_SuperB_CE
CHTC-glidein
CHTC and BATLAB OSG Mirror
GLOW-CHTC VOMS server
GLOW-ATLAS
GLOW-ATLAS-SRM
VDT-ITB-MADISON
VDT-ITB
Lehigh_piranha
LBNL_DSD_ITB
oliverlblgov
LBNL_VTB
UC_Teraport
UC_T2DEV_ITB
MWT2_UC
MWT2_X_SE
```

*inactive:*
```
UTA_SWT2_TEST1
RENCI-Engagement
OSG_US_ASU_OCTILLO
Arizona_CE
GCVO_UIS_CE
GCVO_PUJ_CE_TEST
GCVO_PUJ_CE
NOvA_StashCache_Server
GCVO_UAM_CE
OSG_US_OSU_COWBOY
OSG_TEST_CE
GCVO_UAC_CE
OSG_US_USD_LEGACY
GCVO_UPB_CE
brown-cms-gen3
GCVO_UC_CE
GCVO_UNIANDES_CE
GCVO_UCALDAS_CE
perfSONAR_IllinoisHEP
perfSONAR_GT_testbed_latency
UCSD_GlideinWMS_Submit_1
UCSD_GlideinWMS_Submit_3
UCSD_GlideinWMS_Submit_2
GCVO_UV_CE
osg-ce
MWT2_IU
FNAL_WILSON_2
USCMS-FNAL-WC1-SE2
USCMS-FNAL-WC1-OPP1
NET2-BU
NET2-HU
ANLASC Squid
PNNL_CE00
UC_ATLAS_MWT2
GCVO_UTB_CE
GCVO_UDEM_CE
```

Note that none of these appeared in the list of `OIM_Resource` names from GRACC from the past year.  However, if we use the `SiteName` field from GRACC instead, the following get marked cannot remove:

```
FZU_NOVA
Purdue-Hadoop-HTCE
FIFEBATCH2
FNAL_GPGRID_5
FNAL_GPGRID_4
USCMS-FNAL-WC1-OPP1
```

File: topology/University of Florida/UF CMS Tier2/UFlorida-PG.yaml [hunk 3]
```diff
@@ -75,89 +36,4 @@ Resources:
...
-    ID: 458
-    Services:
-      SRMv2:
-        Description: SRM V2 Storage Element
-        Details:
-          hidden: false
-          uri_override: srm1.ihepa.ufl.edu:8443
-    VOOwnership:
-      CMS: 100
-  perfSONAR_UF:
```
Comment (brianhlin) [inline topology/University of Florida/UF CMS Tier2/UFlorida-PG.yaml]: We actually need to keep this resource due to #110

File: topology/University of Florida/UF CMS Tier2/UFlorida-PG.yaml [hunk 4]
```diff
@@ -36,4 +75,89 @@ Resources:
...
+    ID: 458
+    Services:
+      SRMv2:
+        Description: SRM V2 Storage Element
+        Details:
+          hidden: false
+          uri_override: srm1.ihepa.ufl.edu:8443
+    VOOwnership:
+      CMS: 100
+  perfSONAR_UF:
```
Comment (brianhlin) [inline topology/University of Florida/UF CMS Tier2/UFlorida-PG.yaml]: We only need to keep `perfSONAR_UF`, you can still remove the other ones.

**Comment by edquist** [comment]
@brianhlin & @matyasselmeci, let me know if/when you guys are all happy with everything.

Let me know if you want to rebase-out the Purdue-Hadoop-SE2 back'n'forth, otherwise feel free to go ahead and click the big green merge button.

**Comment by brianhlin** [comment]
We're still sorting out OSU but once we get an answer on that, I vote rebase

**Comment by brianhlin** [comment]
I just got the approval from the OSU admins to remove the `T3_US_OSU` resource. Shawn also gave approval for the two inactive perfsonar services so let's make the OSU change, rebase, merge master, and we should be able to get this in.

**Comment by matyasselmeci** [comment]
Carl, since you're rebasing anyway, you should rebase onto master instead of merging master. It'll make the history cleaner.

**Comment by edquist** [comment]
Yeah I've squashed all the commits on top of master, I'll leave it to @brianhlin and/or @matyasselmeci for final approval & merge.

**Pull request merged**